src/test/RequestQueue.test.ts

```diff
@@ -218,12 +218,79 @@ describe("test RequestQueue", () => {
         { key: "b", requestAction: work },
       ];
       const start = Date.now();
-      const promises = rq.addRequests(requests, delayMs);
+      const promises = rq.addRequests(requests, false, delayMs);
```

Comment:
Nit: Seeing this makes me think it would be nice to have an enum for the request type (HIGH or LOW), since there's no context for it when seeing it without Intellisense... Very likely unnecessarily verbose though.